style_guides/accessibility_guide.md

```diff
@@ -0,0 +1,108 @@
+# Accessibility (A11Y) Guide
+
+This document provides some technical guidelines how to prevent several common
+accessibility issues.
+
+## Interactive elements
+
+### Use `<button>` and `<a>`
+
+**TL;DR** *Use `<button>` and `<a>` instead of click listeners on other elements
+and style it whatever way you need.*
+
+If you want to make an element clickable, use a `<button>` or `<a>` element for it.
+Use a *button* whenever it causes an action on the current page, and an *a* if it
+navigates to a different page. You can use click listeners just fine on these elements.
```

Comment:
> You can use click listeners just fine on these elements.

I think we should advise using a click handler on `button` only, and an `href` with the anchor tag. If we use an anchor with a click handler, then we'll need to apply `kbn-accessible-click` or `KuiKeyboardAccessible`, because browsers won't trigger the click handler when you hit ENTER.